metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -408,7 +408,6 @@ object Embedded {
       .map(_.toPath())
   } catch {
     case NonFatal(e) =>
-      scribe.error(s"Error downloading $dep", e)
```

Comment:
Why remove this one? It's a fallback, so we should not end up here and something might be wrong. Though might be good to change to warning

Reply:
Whoops! That was a remnant from a previous attempt (rather than list out opt-out versions, just have the download logic ignore the error. I ultimately decided this was trying to be too clever.

Anyways, added the line back.